lib/bundler/cli/add.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+require "bundler/cli/common"
+
+module Bundler
+  class CLI::Add
+    def initialize(options, gem_name)
+      @gem_name = gem_name
+      @options = options
+      @options[:group] = @options[:group].split(",").map(&:strip) if !@options[:group].nil? && !@options[:group].empty?
+    end
+
+    def run
+      version = @options[:version].nil? ? nil : @options[:version].split(",").map(&:strip)
+
+      begin
+        dependency = Bundler::Dependency.new(@gem_name, version, @options)
+      rescue Gem::Requirement::BadRequirementError => e
```

Comment:
btw, this constant doesn't exist on old rubygems versions, hence the error on CI